ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -815,8 +811,8 @@ def test_ipacertnsstrust_check(self, modify_cert_trust_attr):
         for Server-Cert
         """
         error_msg = (
-            "Incorrect NSS trust for {nickname} in {dbdir}. "
-            "Got {got} expected {expected}."
+            "Incorrect NSS trust for Server-Cert cert-pki-ca. "
+            "Got CTu,u,u expected u,u,u"
```

Comment:
This line should not be changed upstream, please see https://github.com/freeipa/freeipa-healthcheck/commit/1188d5beaf4a0fdfe98608d9259532b860dabeb1